model/assembly/AssemblyFilesReaderInterface.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2019  (original work) Open Assessment Technologies SA;
+ */
+
+namespace oat\taoDeliveryRdf\model\assembly;
+
+use Generator;
+use tao_models_classes_service_StorageDirectory;
+
+interface AssemblyFilesReaderInterface
+{
+    const SERVICE_ID = 'taoDeliveryRdf';
+
+    /**
+     * @param tao_models_classes_service_StorageDirectory $directory
+     *
+     * @return Generator In format $filePath => StreamInterface
+     */
+    public function getFiles(tao_models_classes_service_StorageDirectory $directory);
+}
```

Comment:
why don't you use extra line at the end of file? 
https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file

Reply:
IDE configuration. Fixed.